backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/FunctionInlining.kt

```diff
@@ -168,14 +168,13 @@ internal class FunctionInlining(val context: Context): IrElementTransformerWithC
         inlineFunctions[functionDescriptor] = childIsBad.value
         data.value = data.value or childIsBad.value
 
-        val currentCalleeIsBad = argsAreBad.value or childIsBad.value or callee.second
+
+        val currentCalleeIsBad = false
```

Comment:
Do we still need this workaround with `Ref<Boolean>`?

Reply:
I'd consider to add TODO about it

Reply:
done